citations-impl/impl/src/java/org/sakaiproject/citation/impl/BaseCitationService.java

```diff
@@ -3062,6 +3072,15 @@ public void saveCitation(Citation citation)
 			save(citation);
 		}
 
+		/*
+		 * (non-Javadoc)
+		 *
+		 * @see org.sakaiproject.citation.api.CitationCollection#saveCitationCollectionOrder(org.sakaiproject.citation.api.CitationCollectionOrder)
+		 */
+		public void saveCitationCollectionOrder(CitationCollectionOrder citationCollectionOrder)
```

Comment:
The citation collection order isn't "part" of a collection.  A collection is made up of sorted citations.  The order the citations are in that make up a collection is determined by a sorted set of citation ids (which is also in the collection).

An example of where you'd want to save a citation collection order that wasn't part of the collection  is if you had a reading list with one HEADING1 and no citations.  Then the 'collection' would have nothing in it, i.e., no citations.  But if you duplicated the reading list, then you'd want to copy the HEADING1 and so you'd want to save a citation collection order (CitationCollectionOrder).  

To clarify, in the code, there are nested and unnested 'collections' which contain just nested or unnested citations which are in a particular order.  These both have their uses.  On top of that, there is also a tree structure representing a nested reading list which contains a top node and children (H1s, H2s, and CITATIONs etc)  which is used for display the nested list.